docs/book/src/topics/workload-identity.md

```diff
@@ -0,0 +1,102 @@
+# Workload Identity
+
+Azure AD Workload identity is the next iteration of Azure AD Pod identity 
+that enables Kubernetes applications (e.g. CAPZ ) to access Azure cloud 
+resources securely with Azure Active Directory.
+
+This document describes a quick start guide of using workload identity and 
+assumes that you have access to Azure cloud.
+
+Workload identity is currently worked upon and cloud provider azure 
+integration is in progress. Please refer [this](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/3589) issue for more details.
+For more details around it, please refer the [proposal](https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/main/docs/proposals/20221611-workload-identity-integration.md)
+
+## Workload Identity Quick Start Guide
+
+- Create a public and private key pair. For example, you can generate the 
+  key pairs using OpenSSL.
+
+  Generate a private key called sa.key using the following command:
+```bash
+$openssl genrsa -out sa.key 2048
+```
+  Generate a public key using the private key. 
+```bash
+$openssl rsa -in sa.key -pubout -out sa.pub
+```
+
+- Create and upload Discovery and JWKS document using this [link](https://azure.github.io/azure-workload-identity/docs/installation/self-managed-clusters/oidc-issuer.html)
+
+- Create a federated identity credential using the steps outlined [here](https://azure.github.io/azure-workload-identity/docs/topics/federated-identity-credential.html)
```

Comment:
In this step, which service account should we be using? Should we create a new service account for this step?

Reply:
Let me clarify this step a bit more. So actually this link of creating Federated identity credential is bit more involved and  I am thinking to write the simplest steps here so that one can get started and also provide the link for more curious users.